Documentation/tutorial/walkthrough_create_a_docfx_project.md

```diff
@@ -0,0 +1,120 @@
+Walkthrough Part I: Generate a Simple Documentation Website
+===================================
+
+By completing this walkthrough, you'll become familiar with the workflow of `docfx` and the general principle of organizing documents inside `docfx`. You will finish this walkthrough with a static website that can be published to any host service.
+
+Step1. Setup docfx
+------------------------
+Download *docfx* from http://dotnet.github.io/docfx/. [Getting Started with docfx](http://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-exe-directly) describes 3 ways to install *docfx*. This walkthrough uses the first one: use *docfx.exe* directly.
+
+1. Download *docfx.zip* and unzip it to `D:\docfx\`
+2. Add `D:\docfx\` to `PATH` so that command `docfx` and be directly called from everywhere for convenience.
+
+Step2. Init a docfx project
+---------------------------
+1. Create a new folder `D:\docfx_walkthrough`  
+2. Start Command Line under `D:\docfx_walkthrough`
+3. Call `docfx init -q`. This command generates a `docfx.json` file under the same folder. `docfx.json` is the configuration file `docfx` uses to generate documentation. `-q` option means generating `docfx.json` quitely using default value, you can also try `docfx init` and follow the instructions to provide your own settings.
+
+Step3. Create the homepage for our website
+------------------------------------------
+Create a file named `index.md` and add some content to it. `index.md` will be the homepage for our website.
+
+```md
+Welcome to the world of docfx!
+=============================
+
+Hello *everyone*.
+```
+
+Step4. Build our website
+-----------------------
+Run command `docfx build`. Note that a new subfolder `_site` is generated under the same folder. This is where the static website is generated.
+
+Step5. Preview our website
+-------------------------
+The generated static website can be published to github pages, azure websites, or your own hosting service without any further changes. You can also run command `docfx serve _site` to preview the website locally.
+
+If port `8080` is not in use, `docfx` will host `_site` under `http://localhost:8080`. If `8080` is in use, you can use `docfx serve _site -p <port>` to change the port to be used by `docfx`.
+
+Congrats! You can now see a simple website similar to:
+![Homepage](images/walkthrough_simple_homepage.png)
+
+----------------------
+
+Step6. Add more articles to the website
+-------------------------
+1. Create subfolder `articles` and `images` under `D:\docfx_walkthrough`
+2. Place a `.md` file to `articles`, e.g. `introduction.md`. If the file references to any resources, put the resources into `images` folder.
```

Comment:
also package the the walkthrough project to a zip file so that user can download.